### PR TITLE
Select option by text

### PR DESCRIPTION
**Pull request by spolischook**
Select option by text if value not match even if value exists

### CODE REVIEW

**Comment by spolischook** [comment]
This version will work with this kind of select:
```html
<select name="priority">
    <option value="low">Low</option>
    <option value="normal">Normal</option>
    <option value="high">High</option>
</select>
```
with:
```php
$selectElement->setValue('High');
```

**Comment by spolischook** [comment]
Or another common example:
```html
<select name="interval_unit">
    <option value="M">minutes</option>
    <option value="H">hours</option>
    <option value="D">days</option>
    <option value="W">weeks</option>
</select>
```
with:
```php
$select->setValue('days');
```

**Comment by spolischook** [comment]
Above examples works with new version of optionQuery. No BC broken

**Comment by spolischook** [comment]
See test results for old optionQuery

File: tests/Custom/SelectTest.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Tests\Driver\TestCase;
+
+class SelectTest extends TestCase
```
Comment (aik099) [inline tests/Custom/SelectTest.php]: As I've mentioned in #241 the test should go into `/driver-testsuite/tests/Form/SelectTest.php` file and be submitted as Mink repo PR.

For example looking as Zombie driver code I can see, that in there setting value by option text is also not supported.

Comment (spolischook) [inline tests/Custom/SelectTest.php]: ok

Comment (spolischook) [inline tests/Custom/SelectTest.php]: Moved to Mink https://github.com/minkphp/Mink/pull/710

**Pull request closed without merging**